trimmer_decorator.rb

```diff
@@ -1,6 +1,9 @@
 require './correct_nameable'
 class TrimmerDecorator < CorrectNameable
   def correct_name
-    @person.correct_name.strip
+    name = @person.correct_name.strip
+    return name[0..10] unless name.length < 10
```

Comment:
- [ ] The Trimmer Decorator needs to make sure that the output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word. currently the output has eleven characters "Maximilianu" you should take name[0..9] and that will give the correct output which is "Maximilian"